bounties/other/age-gender-estimation/1/README.md

```diff
@@ -0,0 +1,13 @@
+# Description
+This is a Keras implementation of a CNN for estimating age and gender from a face image [1, 2]. In training, the IMDB-WIKI dataset is used. This package is vulnerable to Man in the Middle (MITM) attacks due to downloading resources over an insecure protocol. It is possible for an attacker to intercept and alter the packages which may leads to RCE / other attacks which may help the attacker gain access to host.
+
+# Lines :
+* https://github.com/yu4u/age-gender-estimation/blob/6af6148f6eb09bf16e7164490730c71c61c89f3e/lap/create_lap_dataset.py#L74

```

Comment:
Always use `permalinks` of files in the `master` branch, like https://github.com/yu4u/age-gender-estimation/blob/master/lap/create_lap_dataset.py#L74 :smile: